NuGet.Config

```diff
@@ -3,5 +3,6 @@
   <packageSources>
     <clear />
     <add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" />
+    <add key="dotnet9" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet9/nuget/v3/index.json" />
```

Comment:
we should be able to use public nuget, like in id web:

[microsoft-identity-web/NuGet.Config at master · AzureAD/microsoft-identity-web (github.com)](https://github.com/AzureAD/microsoft-identity-web/blob/master/NuGet.Config#L5)